sdks/python/apache_beam/runners/worker/sdk_worker.py

```diff
@@ -38,82 +37,147 @@
 
 
 class SdkHarness(object):
+  REQUEST_METHOD_PREFIX = '_request_'
+
+  def __init__(self, control_address, pipeline_options=None):
+    def _get_worker_count(pipeline_options):
```

Comment:
- Could you add a comment about what is happening here? 
- Also Consolidate terms like `worker_count`, `worker_threads` etc. 
- And make it very clear this worker thread selection is experimental and could (or will) be removed